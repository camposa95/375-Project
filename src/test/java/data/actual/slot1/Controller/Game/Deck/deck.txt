DevCards: [KNIGHT, KNIGHT, VICTORY, PLENTY, VICTORY, ROAD, KNIGHT, KNIGHT, KNIGHT, KNIGHT, KNIGHT, KNIGHT, KNIGHT, ROAD, VICTORY, KNIGHT, PLENTY, VICTORY, KNIGHT, KNIGHT, KNIGHT, MONOPOLY, KNIGHT, VICTORY, MONOPOLY]
