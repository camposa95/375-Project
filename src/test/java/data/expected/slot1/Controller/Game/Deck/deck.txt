DevCards: [ROAD, KNIGHT, ROAD, PLENTY, VICTORY, MONOPOLY, VICTORY, KNIGHT, KNIGHT, KNIGHT, VICTORY, PLENTY, KNIGHT, KNIGHT, VICTORY, KNIGHT, KNIGHT, KNIGHT, MONOPOLY, KNIGHT, KNIGHT, KNIGHT, KNIGHT, VICTORY, KNIGHT]
